src/components/Menu/Menu.constants.js

```diff
@@ -0,0 +1,44 @@
+const menu = {
+  categories: [
+    {
+      name: "tatli",
+      items: [
+        {
+          name: "su",
+          price: "9",
+        },
+        {
+          name: "lo",
+          price: "1",
+        },
+      ],
+    },
+    {
+      name: "icecek",
+      items: [
+        {
+          name: "su",
+          price: "9",
+        },
+        {
+          name: "sldsassjdflo",
+          price: "1",
+        },
+      ],
+    },
+    {
+      name: "icecek",
+      items: [
+        {
+          name: "su",
+          price: "9",
+        },
+        {
+          name: "sldsassjdflo",
+          price: "1",
+        },
+      ],
+    },
+  ],
+};
+export default menu;
```

Comment:
we should not use export default here. This file can have more than 1 constant. Use export const for the menu variable.